include/range/v3/detail/config.hpp

```diff
@@ -389,6 +389,14 @@ namespace ranges
 #define RANGES_RE_ENABLE_WARNINGS
 #endif
 
+
+#if __has_attribute(no_sanitize)
+#define RANGES_INTENDED_UNSIGNED_OVERFLOW \
```

Comment:
_@CaseyCarter critiques @EricWF's pedanticness. It's super effective._

From the wiki for [Integer Overflow](https://en.wikipedia.org/wiki/Integer_overflow):
> In computer programming, an integer overflow occurs when an arithmetic operation attempts to create a numeric value that is outside of the range that can be represented with a given number of bits.
> ...
> The most common result of an overflow is that the least significant representable bits of the result are stored the result is said to wrap around the maximum (i.e. modulo power of two).

It would appear that commonly accepted definitions for Integer overflow consider the modular arithmetic case. But at this point we're just arguing semantics, and I agree with your overall goal of further separating the lingo for well-defined overflow and undefined overflow.